app/controllers/studios_controller.rb

```diff
@@ -0,0 +1,8 @@
+class StudiosController < ApplicationController
+
+  def index
+    @studios = Studio.all
+    @movies = Movie.all
+  end
```

Comment:
All you need in this controller action is line 4, `@studios = Studio.all`. Line 5 isn't required because in the view we are not getting all of the movies in the database, only the movies of this studio, so we would be making use of the association on the `Studio` model. (ie. `studio.movies`)